actions/diagnostics_actions.jsx

```diff
@@ -57,7 +57,14 @@ export function mark(name) {
  *
  */
 export function measure(name1, name2) {
-    if (!global.mm_config.EnableDeveloper === 'true') {
+    const performanceMethodsUsed = [
```

Comment:
Do we need to check for the individual functions, or is it sufficient to check if `window.performance` is set?

Reply:
Not sure if we *need* to. MDN docs do this and the compatibility tables aren't up-to-date so I can't say for sure without a trying on various machines. I generally don't really worry about performance in development mode.